spec/unit/facter/postfix_version_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'spec_helper'
+
+describe Facter::Util::Fact.to_s do
+  before(:each) do
+    Facter.clear
+  end
+
+  describe 'postfix_version' do
```

Comment:
If you changed the subject (as recommended above), then there's no need for that `describe` section, you can just carry on with the `context` below

Reply:
Finally, I'd add a context for when `postconf` is not installed:

```ruby
  context 'when postconf is absent' do
    before :each do
      Facter::Util::Resolution.stubs(:which).with('postconf').returns('')
    end
    it { expect(subject.value).to be_nil }
  end
```